src/screens/ProfileScreen.js

```diff
@@ -74,7 +74,7 @@ export default function ProfileScreen({ navigation }) {
 
   return (
     <ProfileScreenPage testID="profile-screen">
-      <Header justify="space-between">
+      <Header justify="space-between" style={{ alignItems: 'center' }}>
```

Comment:
@skylarbarrera hehe sorry to be a loser buuut

u should make this `<Header align="center" justify="space-between">` 😘😘😘😘😘😘